src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureTableStorage.cs

```diff
@@ -522,5 +451,54 @@ public bool DecodeException(Exception e, out HttpStatusCode httpStatusCode, out
         {
             return AzureStorageUtils.EvaluateException(e, out httpStatusCode, out restStatus, getRESTErrors);
         }
+
+        private async Task Init(CancellationToken ct)
+        {
+            this.logger.Info((int)AzureProviderErrorCode.AzureTableProvider_InitProvider, $"AzureTableStorage initializing: {this.options.ToString()}");
```

Comment:
suggest to use `ILogger.LogInformation` instead. that's their native method. `ILogger.Info` extension exists mainly to make the migration to MEL easier.